servers.json

```diff
@@ -1411,5 +1411,19 @@
         "addresses": [
             "ymzc.pro"
         ]
+    },
+    {
+        "id": "wibn",
+        "name": "WiBn,
```

Comment:
Fix your json formatting and upload your server logos

Reply:
Added the server logo and _hopefully_ fixed the json formatting

Reply:
Pretty sure the stuffs work now.